articles/cli.md

```diff
@@ -0,0 +1,178 @@
+## Bonsai Command-Line Interface (CLI)
+
+Bonsai CLI (Command-Line Interface) allows workflows to be launched from the OS command-line tool. In addition to opening workflows with the default graphic interface, the CLI opens the possibility to use other advanced features, that will be covered ahead, such as "no-editor" mode, initialization of variable's values, and specifying layouts.
+
+
+&nbsp;
+___
+### Installation and basic use
+Bonsai CLI can be called by targeting ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line:
+
+```cmd
+> pathToBonsaiExe\Bonsai.exe
+```
+Alternatively, by adding ```pathToBonsaiExe``` to your System Path, simply:
+
+```cmd
+> Bonsai
+```
+
+This command will open the default Bonsai editor.
+
+___
+&nbsp;
+### Opening a Workflow
+
+To open an existing workflow, ```myWorkflow.bonsai```, use the syntax:
+
+```cmd
+> Bonsai myWorkflow.bonsai
+```
+
+##### Example:
+
+```cmd
+> Bonsai myCameraWorkflow.bonsai
+> Bonsai "C:\Users\User\Desktop\Example\myCameraWorkflow.bonsai"
```

Comment:
is this an example that you are adding to the repo, so that people can actually run it? Otherwise I'm not sure what is the added benefit of having this additional example...

Reply:
Sure, I was debating whether to add it or not since there is no file structure for workflows yet. What was the solution we came up with again? Simply opening a ""workflows" folder?

Reply:
I ended up making the article agnostic to a specific workflow.